index.js

```diff
@@ -425,4 +425,22 @@
     return JSON.parse(s);
   });
 
+  //  regexp  /////////////////////////////////////////////////////////////////
+
+  //. match :: RegExp -> String -> Maybe [Maybe String]
+  //.
+  //. Takes a regular expression and tests it against the supplied string
+  //. returning a Nothing if there are no matches or a Just containing an
+  //. array of Maybe types corresponding to the matches found.
+  //.
+  //. ```javascript
+  //. > S.match(/abcd/, 'abcdefg')
+  //. Just([Just('abcd')])
```

Comment:
Let's use double quotes on this line, as this will be required by #36. Stick with single quotes on the input line, though.